pulp_rpm/app/serializers.py

```diff
@@ -165,6 +168,10 @@ class PackageSerializer(SingleArtifactContentSerializer):
     rpm_header_end = serializers.IntegerField(
         help_text=_("Last byte of the header"),
     )
+    rpm_modular = serializers.BooleanField(
```

Comment:
If possible, it would be nice if this were instead a nullable foreign key pointing to the artifact that it belongs to instead of just a boolean.

Reply:
But, on the other hand, if a user tries to delete a module, we need to know what RPMs to delete, and if they try to delete a modular RPM we need to be able to deny them or delete the related units.  That's a hard problem too.